impl/src/main/java/io/kroxylicious/testing/kafka/testcontainers/TestcontainersKafkaCluster.java

```diff
@@ -224,6 +225,23 @@ private Stream<GenericContainer<?>> allContainers() {
     @SneakyThrows
     public void start() {
         try {
+
+            // Workaround for https://github.com/kroxylicious/kroxylicious-junit5-extension/issues/30
+            // This fix can be removed once https://github.com/testcontainers/testcontainers-java/issues/6667 is resolved.
+            Unreliables.retryUntilSuccess(
+                    3,
+                    () -> {
+                        try {
+                            network.getId();  // Has side effect of creating the network
+                        } catch (Throwable t) {
+                            // close is required to reset an internal atomic boolean,
+                            // object actually remain in usable state.
+                            network.close();
+                        }
+                        return null;
+                    }
+            );
```

Comment:
Can we pull this out as a method, mostly for an intention revealing name, something like `startNetwork`